base/applications/taskmgr/lang/en-US.rc

```diff
@@ -136,7 +136,7 @@ BEGIN
         END
         MENUITEM "Set &Affinity...", ID_PROCESS_PAGE_SETAFFINITY
         MENUITEM SEPARATOR
-        MENUITEM "&Properties", ID_PROCESS_PAGE_PROPERTIES
+        MENUITEM "P&roperties", ID_PROCESS_PAGE_PROPERTIES
```

Comment:
I'm going to say something controversial, but it's more natural to have the '&' on 'P' of "Properties". This would imply changing the "Set &Priority" to something like "Set P&riority", for example. (Other translations do something similar already.)

Reply:
OK.